packages/date-time-utilities/src/dateValues/timeConstants.ts

```diff
@@ -9,4 +9,7 @@ export const TimeConstants = {
   DaysInOneWeek: 7,
   MonthInOneYear: 12,
   HoursInOneDay: 24,
+  SecondsInOneMinute: 60,
+  OffsetTo24HourFormat: 12,
+  TimeFormatRegex: /^(\d\d?):(\d\d):?(\d\d)? ?([ap]m)?/i,
```

Comment:
Can you add a `/** doc comment */` with info about what the groups are? (group 1 is hours, etc)

Reply:
Resolved by adding `/** doc comment */` explaining groups in `TimeFormatRegex`.